internal/runners/clean/run.go

```diff
@@ -12,7 +15,7 @@ func (u *Uninstall) runUninstall() error {
 		return err
 	}
 
-	err = removeInstall(u.installPath)
+	err = removeInstall(u.installDir)
```

Comment:
I think this function could fail on Windows as it would be unable to remove the installation directory if the tray or service is still running. We may need to kill the tray and stop the service before attempting to remove them.

Reply:
Yes, I think I have to merge with your PR first for this to work... :)